content/docs/web-components.md

```diff
@@ -12,20 +12,58 @@ Most people who use React don't use Web Components, but you may want to, especia
 
 ## Using Web Components in React {#using-web-components-in-react}
 
+Web components are just custom (HTML) elements which can be used directly in React:
+
 ```javascript
-class HelloMessage extends React.Component {
-  render() {
-    return <div>Hello <x-search>{this.props.name}</x-search>!</div>;
-  }
-}
+const Dropdown = props => {
+  return <my-dropdown></my-dropdown>;
+};
 ```
 
-> Note:
->
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
```

Comment:
The note about needing a ref is helpful, can we keep it?

Reply:
I can move it to the top again. In case you haven't seen, I added the note to the bottom in a general FAQ section.